### PR TITLE
Add option for login layout background image size

### LINKED ISSUE #2877: Specify background image size for login screen
## Is your feature request related to a problem? Please describe 👀

If not using a gradient, but specifying a background image, it is useful to be able to specify the background size.

Currently:

Screenshot 2024-02-01 at 3 26 44 PM

Desired:

Screenshot 2024-02-01 at 3 32 50 PM

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #2877

# 👩🏻‍💻 What does this PR do? 
 

I had to add this in order to make a demo theme look nice. Just a minor tweak to allow specifying the `background-size` property on the login screen LHS

# 🧪 How has/should this change been tested? 

You can try with the custom theme in [this folder](https://drive.google.com/open?id=1IBM2SmyfQQdRzh341CXONX4rk7Bl6QjZ&usp=drive_fs)

## 📃 Documentation

_No user facing changes_

### CODE REVIEW

File: client/packages/host/src/components/Login/LoginLayout.tsx [hunk 1]
```diff
@@ -40,6 +40,7 @@ export const LoginLayout = ({
         flex="1 0 50%"
         sx={{
           backgroundImage: (theme: Theme) => theme.mixins.gradient.primary,
+          backgroundSize: (theme: Theme) => theme.mixins.gradient.size,
```
Comment (CarlosNZ) [inline client/packages/host/src/components/Login/LoginLayout.tsx:43]:
```suggestion
          backgroundSize: (theme: Theme) => theme.mixins.gradient.size,
          backgroundPosition: 'center',
          backgroundRepeat: 'no-repeat',
```

The "center" makes it look better through a bigger range of window resizes.
The "no-repeat" does nothing with your theme (if "cover" is set) but is better when the "size" property is not specified (i.e. defaulting to "contain"), so makes sense to leave it on permanently. In fact, it's arguable whether this theme might actually be better just with "no-repeat" and not changing the "backgroundSize" (this makes it never overflow the box, even with extreme aspect ratios), but that's subjective and not particularly important in most cases.

**Review by CarlosNZ — APPROVED** [review]
Great thanks. Just made an additional suggestion for a couple of other properties that probably help.

**Comment by CarlosNZ** [comment]
Also -- it would be nice if you could toggle the "Custom Theme" and "Custom Icon" buttons off and on again without losing the current customisations. Perhaps "customTheme" enabled setting could be a separate local storage item to the theme data itself? Not huge, but might be worth changing at some point if it's easy to do.

**Comment by mark-prins** [comment]
> it would be nice if you could toggle the "Custom Theme" and "Custom Icon" buttons off and on again without losing the current customisations

Yeah - and I started down that path.. but the current implementation stores only the value and the hash in the database. To add the enabled prop requires api changes so I'll leave that for a separate issue.

**Pull request merged**